hashing/src/chunk_with_proof.rs

```diff
@@ -0,0 +1,304 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use casper_types::bytesrepr::{self, Bytes, FromBytes, ToBytes};
+
+use crate::{error::MerkleConstructionError, indexed_merkle_proof::IndexedMerkleProof, Digest};
+
+/// Represents a chunk of data with attached proof.
+#[derive(PartialEq, Debug, JsonSchema, Serialize, Deserialize)]
+#[schemars(with = "String", description = "Chunk of data with attached proof.")]
```

Comment:
This line should be removed: it's telling schemars to use the schema for `String` rather than deriving the proper schema for this struct.  The schema description will be automatically pulled from the doc comment for the struct too.

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/2119/commits/bec1fdff1eb1ea8bf8388d9a7626352c1427b44d